package.json

```diff
@@ -59,6 +59,8 @@
     "test:e2e:headed": "npx playwright test --headed",
     "test:e2e:dist": "lerna run test:e2e:dist --stream",
     "test:e2e:serve": "yarn test:data && lerna run test:e2e:serve --stream",
+    "test:e2e:fast": "lerna run test:e2e:fast --stream",
+    "test:data:serve": "yarn test:data && cd testdata && dicomwebserver -c ./static-dicomweb.json5 -p 3030",
```

Comment:
And checkout/run the dicomwebserver locally instead of incorporating that data into the rsbuild